analyzers/LdapQuery/LdapQuery.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Ldap_Query",
+  "version": "2.0",
+  "author": "Florian Perret @cyber_pescadito",
+  "url": "https://github.com/cyberpescadito/Cortex-Analyzers/tree/master/analyzers/LdapQuery",
+  "license": "AGPL-V3",
+  "description": "Query your LDAP server to harvest informations about an user of your organization",
+  "dataTypeList": ["username", "mail"],
+  "command": "LdapQuery/LdapQuery.py",
+  "baseConfig": "LdapQuery",
+  "configurationItems": [
+    {
+        "name": "LDAP Server Address",
```

Comment:
I would recommend using the "Snake Case" notation: `xxx_yyyy_zzzz` for all the configuration items' name